README.md

```diff
@@ -6,23 +6,44 @@
 
 ## Overview
 
-Software Discovery Tool is a tool for discovering open source software for zArchitecture/s390x for any Z operating system. The data for the software, such as their names and versions available, are contained in this repository. Please refer to the documentation in [this](https://github.com/openmainframeproject/software-discovery-tool) repo to learn more about the data directory structure.
+Software Discovery Tool is a comprehensive resource for discovering open-source software tailored for zArchitecture/s390x across various Z operating systems. This repository hosts essential data such as software names and their availability, facilitating seamless exploration and utilization.
+
+🚀 Please refer to the documentation in [this](https://github.com/openmainframeproject/software-discovery-tool) repo to learn more about the data directory structure.
```

Comment:
Since emojis are not universally supported in terminal clients and often require additional packages if they are, we should probably avoid using them in our README, sorry! I like them too :smile:

Reply:
I too like them! 😊 No problem, I have updated accordingly.